recipes/hatch-docstring-description/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hatch-docstring-description" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hatch_docstring_description-{{ version }}.tar.gz
+  sha256: a21da39911023a3af0a5ec7b2668393111776b420758e89eedc34ecd07ba4afd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python >=3.8
+    - hatchling
+
+test:
+  imports:
+    - hatch_docstring_description
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/flying-sheep/hatch-docstring-description
+  summary: Hatch plugin to derive package description from Python package docstring
+  license: GPL-3.0
```

Comment:
The linter is correct. This must either be `GPL-3.0-or-later` or `GPL-3.0-only`.

Reply:
well, that’s an actionable thing, since I’m the maintainer and can say what it meant.